### DISCUSSION TITLE
Change navigation keys h,j,k,l vim mode

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [aristocratos/btop#991] [Q&A]
How can I change the navigation from `h,j,k,l` to `h,a,e,i`? It's because I use an angled [version](https://gist.githubusercontent.com/aljustiet/3402bbe0e3aa1cc7fef11977fddfe364/raw/f7ebb2e032050e8b27b479de30461eeb7a16dfd4/gistfile1.txt) of an alternative keyboard layout called [graphite](https://github.com/rdavison/graphite-layout).

**Comment by azizsayfiddin** [comment]
I couldn't find it in the man pages.

**Comment by deckstose** [comment]
This is not possible.

**Reply by azizsayfiddin** [reply]
Software-wise? How much effort would it take implement this?

**Reply by deckstose** [reply]
It's just not implemented :) Probably a lot if you'd want the user to choose custom keys, since it would required a check for other bound keys (if you disregard this, pretty trivial) and if you want to hardcode those 4 keys this won't flow since we can't maintain every users preferred keybindings
